### PR TITLE
rpm centos: use systemctl with systemd platform

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: packages/rpm/centos/mariadb-10.1-mroonga.spec.in [hunk 1]
```diff
@@ -5,8 +5,12 @@
 %define mariadb_download_base_url_default https://downloads.mariadb.org/f
 %if %{_centos_ver} == 7
 %define mariadb_dist_default    el7.centos
+%define service_status() (systemctl status %1 > /dev/null)
```
Comment (kou) [inline packages/rpm/centos/mariadb-10.1-mroonga.spec.in]: How about removing `> /dev/null` from macro?
It's common in both `systemctl` version and `/sbin/service` version.

Comment (kou) [reply packages/rpm/centos/mariadb-10.1-mroonga.spec.in]:
```shell
 if ! %{service_status mysqld} > /dev/null; then
```

Comment (kenhys) [reply packages/rpm/centos/mariadb-10.1-mroonga.spec.in]: Thanks, unified to use `> /dev/null` style for consistency.

Comment (kou) [reply packages/rpm/centos/mariadb-10.1-mroonga.spec.in]: Really?
The diff still has `> /dev/null` in macro.

Comment (kenhys) [reply packages/rpm/centos/mariadb-10.1-mroonga.spec.in]: Thanks, still `> /dev/null` exists... this is not intended, so I've fixed.

File: packages/rpm/centos/percona-server-57-mroonga.spec.in [hunk 2]
```diff
@@ -18,6 +18,14 @@
 
 %define groonga_required_version @REQUIRED_GROONGA_VERSION@
 
+%if %{_centos_ver} == 7
```
Comment (kou) [inline packages/rpm/centos/percona-server-57-mroonga.spec.in]: CentOS 8 will release soon.
So `${_centos_ver} >= 7` is better.

**Pull request merged**